weblate/addons/base.py

```diff
@@ -148,20 +148,28 @@ class UpdateBaseAddon(BaseAddon):
     It hooks to post update and commits all changed translations.
     """
     events = (EVENT_POST_UPDATE, )
-    message = '''Update translation files
+    message = '''Update translation files{component}
 
-Updated by {name} hook in Weblate.'''
+Updated by "{name}" hook in Weblate.'''
```

Comment:
Maybe it would be better to use same rendering engine and style as other Weblate commit messages?

```
Update translation files

Updated by "{{ hook_name }}" hook in Weblate.

Translation: {{ project_name }}/{{ component_name }}
Translate-URL: {{ url }}
```